### ISSUE TITLE
Mimmick game features

### ISSUE THREAD

**Issue by ZooterTheWooter** [Laritello/osu-music#106]
Please add a Double Time button or Nightcore button, some songs are only fun to listen in nightcore mode because they're really slow paced. 

It would also be really cool if you could figure out a way for the application to read the highest difficulty songs and have a button where it plays the clicking noise from the circles in game. 

Talking about this hit sound : https://www.youtube.com/watch?v=HgUCxmzMjNk 

example here : https://imgur.com/a/ScGQ6zF 

I'm a retired player so I plan on using this software since I can't physically play anymore, its really nice software being able to listen to the music without having to launch into the game itself and scroll through the menus constantly. But these 2 features would be really nice to have because it just sounds wrong sometimes not being able to listen to some songs with out the hit circle sounds.  

edit : also just came up with another idea, please have an option where we can click on a "copy to clipboard" option so if we wanna look up the song on Spotify or youtube it saves us the trouble of having to keep looking at the name.

**Comment by Laritello** [comment]
Yeah, I tried to implement something like this earlier. Wasn't able to achieve it in satisfying manner, so decided against it. I'll look at it again in free time, but can't promise I implement it. osu.Music mostly works directly with osu database which is faster and easier than manually work with beatmap files. And for me to achieve this funcionality I have to work with beatmap files.

**Comment by ZooterTheWooter** [comment]
So doing a nightcore feature wouldn't be possible? NightCore or DT feature? It is just the game speeding up at 1.5x speed. So it should be possible.

**Comment by Laritello** [comment]
Sorry for the confusion. I was talking about the hit sound feature. As for DT/NC: used audio library doesn't support playback speed change out of the box, but I should be able to implement it myself.

**Comment by ZooterTheWooter** [comment]
Sweet, I also did have a couple other requests I just thought of. 
https://imgur.com/a/9logUzs can you add a beatmap preview window? So we can see what version of a song is? Since there's like 20 different versions of certain beatmaps now. Would be nice.

https://imgur.com/a/rfY3ihb also this one is a must, when hovering over the osu,music app its annoying I cant pause it without having to open it up or alt tabbing into the program. Can you please add a small play button like what windows media player does? 

Many thanks!
